error_reporting/google/cloud/error_reporting/_gapic.py

```diff
@@ -14,53 +14,50 @@
 
 """GAX wrapper for Error Reporting API requests."""
 
-from google.cloud._helpers import make_secure_channel
-from google.cloud._http import DEFAULT_USER_AGENT
+from google.api_core.gapic_v1 import client_info
 
-from google.cloud.gapic.errorreporting.v1beta1 import (
+from google.cloud.error_reporting import __version__
+from google.cloud.errorreporting_v1beta1.gapic import (
     report_errors_service_client)
-from google.cloud.proto.devtools.clouderrorreporting.v1beta1 import (
+from google.cloud.errorreporting_v1beta1.proto import (
     report_errors_service_pb2)
 from google.protobuf.json_format import ParseDict
 
-from google.cloud.error_reporting import __version__
+_CLIENT_INFO = client_info.ClientInfo(
+    client_library_version=__version__)
 
 
 def make_report_error_api(client):
-    """Create an instance of the GAX Logging API.
+    """Create an instance of the gapic Logging API.
```

Comment:
Is this an auto-gen bug that needs fixing? I assume this is manual.

Reply:
This is al manual stuff.